code/datums/status_effects/buffs.dm

```diff
@@ -188,15 +191,52 @@
 		return
 	REMOVE_TRAIT(owner, TRAIT_CHUNKYFINGERS, VAMPIRE_TRAIT)
 	var/mob/living/carbon/human/H = owner
-	H.physiology.brute_mod /= 0.5
-	H.physiology.burn_mod /= 0.8
+	H.color = original_color
+	H.physiology.brute_mod /= 0.4
+	H.physiology.burn_mod /= 0.5
 	H.physiology.stamina_mod /= 0.5
 	H.physiology.stun_mod /= 0.5
 	if(bonus_damage_applied)
 		bonus_damage_applied = FALSE
 		H.physiology.melee_bonus -= 10
 		H.dna.species.punchstunthreshold -= 8
 
+/datum/status_effect/vampire_gladiator
+	id = "vampire_gladiator"
+	duration = 30 SECONDS
+	tick_interval = 1 SECONDS
+	alert_type = /obj/screen/alert/status_effect/vampire_gladiator
+	var/healing_ticks_left // We use this to see how long we can keep healing the vampire
```

Comment:
why is this var needed? surely it will only apply 30 tickets of heal anyway because after 30 seconds the buff goes away. and it applies a tick every second.